posthog/migrations/0918_add_sdk_config_team_setting.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.26 on 2025-11-23
+
+from django.db import migrations, models
+
+import posthog.models.utils
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0917_alter_survey_linked_insight"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="team",
+            name="sdk_config",
+            field=models.JSONField(blank=True, null=True, validators=[posthog.models.utils.validate_sdk_config]),
```

Comment:
@webjunkie i'm super out-of-date with the django migration stuff  
  
do i need to worry about the `null`​ here?  
it'd be great to allow `null`​ since it' valid not to have a setting but :see_no_evil:

Reply:
Sure, `null` is better than `not null`.

However, we are still adding a field to the team table. Last time someone did that, we had an incident cause there was contention. Not sure if it works if you deploy this during low traffic hours.

https://posthog.slack.com/archives/C09QMRRV8EA

Would it make sense to add the config to another table, and have a onetoone key to team?

Reply:
> I'm not sure of the context I have here, regarding Rust.

same, that's my problem... i'd have to discover the consequences of an otherwise reasonable solution by making all the mistakes :sweat_smile:  
  
ok... then i need to figure out the third solution :rolling_on_the_floor_laughing: